code/datums/diseases/japanesefever.dm

```diff
@@ -0,0 +1,92 @@
+/datum/disease/japanesefever
+	name = "Japanese Fever"
+	desc = "Oh. Oh no. Oh no no no."
+	max_stages = 5
+	spread = "On contact"
+	spread_type = CONTACT_GENERAL
+	cure_id = BLEACH
+	agent = "Unknown"
+	affected_species = list("Human")
+	permeability_mod = 1
+	var/revelation_had = 0
+
+/datum/disease/japanesefever/stage_act()
```

Comment:
This flavor text sequence is :100: yikes from me

Reply:
Just the way I like it